scaling_performance/host_practices.adoc

```diff
@@ -0,0 +1,106 @@
+[[scaling-performance-capacity-host-practices]]
+= Recommended Host Practices
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+[[scaling-performance-capacity-host-practices-master]]
+== Recommended Practices for {product-title} Master Hosts
+
+Besides pod traffic, the most-used data-path in an {product-title}
+infrastructure is between the {product-title} master hosts and etcd. The
+{product-title} API server (part of the master binary) consults etcd for node
+status, network configuration, secrets, and much more.
```

Comment:
s/much more/other information 
Not sure the right term, but "and much more" sounds to marketing-speak, like an informercial!